lib/tasks/dev.rake

```diff
@@ -1,3 +1,95 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  p "Creating sample data"
+
+  if Rails.env.development?
+    Category.destroy_all
+    Listing.destroy_all
+    Message.destroy_all
+    Offer.destroy_all
+    User.destroy_all
+  end
+
+  12.times do
+    name = Faker::Name.first_name
+    username = Faker::Name.name
+    address = Faker::Address.full_address
+    bio = Faker::Quote.matz
+    location = Faker::Address.city
+    u = User.create(
+      email: "#{name}@example.com",
+      password: "password",
+      username: name,
+      name: username,
+      address: address,
+      image: "https://robohash.org/#{rand(9999)}",
+      bio: bio,
+      location: location,
+    )
+  end
+
+  3.times do
```

Comment:
As we discussed during our meeting, you can create category names by accessing the enum names that you defined in your Category model. You can update the code to look like this: ```
  categories = []
  Category.names.keys.each do |category_name|
    categories << Category.create(name: category_name)
  end```